src/third_party/progress/progress.hpp

```diff
@@ -208,6 +216,11 @@ class C_Progress_display : public C_Progress
     } // display_tic
 };
 
+inline C_Progress& C_Progress::dummy()
+{
+  static C_Progress null;
```

Comment:
Whqt do you think about (VS compile does not have guarantee to make the static a lock free operation)
```
C_Progress * instance = nullptr;
std::once_flag flag;

inline C_Progress& C_Progress::dummy()
{
  std::call_once(flag, []) { instance = new C_Progress{}; }
  return *instance;
}
```